app/views/DiscussionsView/index.tsx

```diff
@@ -63,7 +63,7 @@ const DiscussionsView = ({ navigation, route }: IDiscussionsViewProps): JSX.Elem
 
 		setLoading(true);
 		try {
-			const result: any = await RocketChat.getDiscussions({
+			const result = await RocketChat.getDiscussions({
```

Comment:
Are you sure the `getDiscussions` method returns the entire `IMessage` object?

Maybe it wasn't clear, but this issue of getting all the endpoints is exactly to fix these cases too.
In this example, we only put `IMessage` as the return because when we migrated the files from the web to the mobile we had to take advantage of some old types.